src/core/search/searchdatatransformer.js

```diff
@@ -18,16 +18,15 @@ import ResultsContext from '../storage/resultscontext';
  * component library and core storage understand.
  */
 export default class SearchDataTransformer {
-  static transform (data, urls = {}, formatters) {
-    let response = data.response;
+  static transformUniversal (data, urls = {}, formatters) {
     return {
-      [StorageKeys.QUERY_ID]: response.queryId,
-      [StorageKeys.NAVIGATION]: Navigation.from(response.modules),
-      [StorageKeys.DIRECT_ANSWER]: DirectAnswer.from(response.directAnswer, formatters),
-      [StorageKeys.UNIVERSAL_RESULTS]: UniversalResults.from(response, urls, formatters),
-      [StorageKeys.INTENTS]: SearchIntents.from(response.searchIntents),
-      [StorageKeys.SPELL_CHECK]: SpellCheck.from(response.spellCheck),
-      [StorageKeys.LOCATION_BIAS]: LocationBias.from(response.locationBias)
+      queryId: data.queryId,
+      navigation: Navigation.fromCore(data.verticalResults),
+      directAnswer: DirectAnswer.fromCore(data.directAnswer),
+      universalResults: UniversalResults.fromCore(data, urls),
+      searchIntents: SearchIntents.fromCore(data.searchIntents),
+      spellCheck: SpellCheck.fromCore(data.spellCheck),
+      locationBias: LocationBias.fromCore(data.locationBias)
```

Comment:
Small thing, but I think we should continue to use `[StorageKeys.blah]` as the keys here. It indicates that an entry is going to be put into storage and where. While it's unlikely that we will actually change any of the `StorageKey` values, we might. If we did, we don't want to have to update the value in multiple places.

Reply:
Sure, i can revert